sonar-php-plugin/src/main/java/org/sonar/plugins/php/phpunit/OverallCoverageResultImporter.java

```diff
@@ -19,14 +19,14 @@
  */
 package org.sonar.plugins.php.phpunit;
 
-import org.sonar.api.batch.fs.FileSystem;
 import org.sonar.api.batch.sensor.coverage.CoverageType;
 import org.sonar.api.measures.CoreMetrics;
+import org.sonar.plugins.php.PhpPlugin;
 
-public class PhpUnitOverallCoverageResultImporter extends PhpUnitCoverageResultImporter {
+public class OverallCoverageResultImporter extends CoverageResultImporter {
 
-  public PhpUnitOverallCoverageResultImporter(FileSystem fileSystem) {
-    super(fileSystem);
+  public OverallCoverageResultImporter() {
+    super(PhpPlugin.PHPUNIT_OVERALL_COVERAGE_REPORT_PATH_KEY, "overall coverage");
     linesToCoverMetric = CoreMetrics.OVERALL_LINES_TO_COVER;
     uncoveredLinesMetric = CoreMetrics.OVERALL_UNCOVERED_LINES;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this use of "OVERALL_UNCOVERED_LINES"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)